components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -971,9 +1118,24 @@ mojom::FeedV2Ptr FeedV2Builder::GenerateAllFeed() {
     base::ranges::move(items, std::back_inserter(feed->items));
   };
 
+  std::vector<std::string> eligible_content_groups;
+  for (const auto& channel_id : subscribed_channels) {
+    eligible_content_groups.push_back(channel_id);
+  }
+  // TODO(LorenzoMnto, fallaciousreasoning): What about direct feeds?
```

Comment:
Direct feeds actually generate a Publisher (with a random GUID ID), so I think this should actually Just Work(tm).